static/scripts/rewards/app-state.ts

```diff
@@ -48,12 +48,12 @@ export class AppState {
     return networkExplorers[this.reward.networkId] || "https://etherscan.io";
   }
 
-  nextPermit(): RewardPermit | null {
+  nextPermit(): Permit | null {
```

Comment:
Why did you change from `RewardPermit`? I think more specific naming is better.

Reply:
Can change the name. `Permit` was called that way in `@ubiquibot/permit-generation`, so have to change it there.